exonum-java-binding/service-archetype/src/main/resources/archetype-resources/src/test/java/ServiceModuleTest.java

```diff
@@ -45,6 +48,19 @@ void testTransactionConverterBinding() {
     assertThat(s, instanceOf(MyTransactionConverter.class));
   }
 
+  @Test
+  void testServiceInstantiation() {
```

Comment:
wouldn't it be better to create a separate class with junit5 extension?

Reply:
Not sure. This particular test is only useful at project initialization to verify the app is installed properly + as an example of testkit configuration :thinking: We can extend some comments here and there/modify the test display name to make their purpose more apparent.

Reply:
I removed the existing tests as they do not bring much value on top of this new one.

Reply:
and documented the likely failure